src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -372,15 +372,15 @@ protected static void leftAlignVariant(final VariantContextBuilder builder, fina
 
         // 1. while changes in alleles do
         while (changesInAlleles) {
+            final int oldStart = theStart;
+            final int oldEnd = theEnd;
 
-            changesInAlleles = false;
             // 2. if alleles end with the same nucleotide then
             if (alleleBasesMap.values().stream()
                     .collect(Collectors.groupingBy(a -> a[a.length - 1], Collectors.toSet()))
                     .size() == 1 && theEnd > 1) {
                 // 3. truncate rightmost nucleotide of each allele
```

Comment:
Can this be turned into a continuation of comment 2? If I understand right, this block just trims the alleles if they end with the same string (e.g. REF = "ACG", ALT="ATG" -> REF = "AC", ALT="AT" ... maybe even put this example in comment?)